docs/faq.md

```diff
@@ -55,7 +60,7 @@ export default {
 };
 ```
 
-### How can I opt-out of Angular ngcc?
+## How can I opt-out of Angular ngcc?
```

Comment:
was the hierarchy level change intentional? or auto-save kind of formatting

Reply:
Yes it was. I streamlined it. It was chaotic.